src/web/web-service/web_server.cpp

```diff
@@ -59,6 +59,40 @@
 
 namespace otbr {
 namespace Web {
+namespace {
```

Comment:
use static instead of anonymous namespace?

Reply:
Done.